source/standards/programming-languages.html.md.erb

```diff
@@ -101,6 +101,33 @@ We used Scala in the early days of GDS. GOV.UK Licensing is the only remaining
 application written in Scala but we've found it hard to support because of a lack
 of skills in GDS. Do not use Scala for new projects.
 
+## Mobile Development
+
+For developing mobile apps, we use:
+- [Swift](https://www.swift.org/) for [iOS](https://developer.apple.com/)
+- [Kotlin](https://kotlinlang.org/) for [Android](https://developer.android.com/).
+
+To give users the expected experience on their respective platform, we prefer to use the native languages over cross-platform solutions. 
+
+### Swift
+
+When starting a new app project, you'll likely want to use the most recent version of Swift which will be installed with Xcode.
+
+The targetted version of iOS should be discussed during planning and investigation as certain iOS features may not be available in earlier versions which could be crucial to your app's behaviour. 
+For example, a good starting point for government apps would be a range of n - 5 or n - 3 (n being the latest major release of iOS).
```

Comment:
Hi, can we remove conversations about the lowest supported iOS version for now? We're doing some work in parallel about what our position is around app version support, and it would be good to make sure that we align and not publish something we don't end up agreeing with?

Reply:
I think we totally need something like this in the GDS Way,
But I think having spoken to @AdamHigginsonGDS it's a bit early for us to make that call (let alone at the GDS level).

I'd recommend @mcgarj - remove this, and as @jonnyandrew has noted remove the android bit.

And let's us start drafting a follow up page that addresses this?

We can then return and link to that from here.

There's big chats to be had there:

- what's the security consequence of supporting older versions (so are we locked to OS support?)
- or do we follow what design system does for browsers, 2% of traffic or more?
- what's the accesibility and universal access implications

I suspect there's big tradeoffs so let's have that debate in full.

Reply:
I'm not sure it's worth having anything around versioning if the general advice is plan according to the project.
We could start with some easily met guidelines, such as: "don't go below 14.0 and always aim to update to support the latest version within X time of its official release." and "when dropping OS versions in the future for anything other than security reasons, always attempt to leave a trailing version on the store for X time to avoid orphaning users with legacy OS versions"
IDV and GOV.UK are good examples of apps potentially weighing security and availability slightly differently.

Reply:
Makes sense to me, and good to see more input on how we can structure this! I've removed the parts for iOS and Android about specific versions, is there anything else that should be changed/added, or are we most comfortable with the one line in each section about likely wanting to use the most recent version of Swift/Kotlin?